integration/deno-compiler-test.ts

```diff
@@ -2,7 +2,7 @@ import { test, expect } from "@playwright/test";
 import * as fse from "fs-extra";
 import path from "path";
 import shell from "shelljs";
-import glob from "glob";
+import glob from "fast-glob";
```

Comment:
Updated for consistency, but it should also make the test a tiiiiiiiiiny bit faster